cmd/tailscale/cli/licenses.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
```

Comment:
2022

Reply:
psh.  I have it on good authority it's [March 907th, 2020](https://covidstandardtime.com/).

Reply:
Legit. It does feel like we never left March 2020.